extensions/ui/org.eclipse.smarthome.ui.classic/ESH-INF/config/config.xml

```diff
@@ -18,6 +18,11 @@
 			<label>Default Sitemap</label>
 			<description>The sitemap to show if no parameter is passed.</description>
 		</parameter>
+		<parameter name="disableHtmlCache" type="text" required="false">
+			<label>Disable HTML cache</label>
+			<description>Disable in-memory caching of html fragments.
+				If this is true, on every request the html files are loaded from disk.</description>
+		</parameter>
```

Comment:
as only "true" and "false" are valid options here, it would be good if you would add option elements here (like for the iconType).